src/Common/Commands.Common.Authentication/Authentication/ManagedServiceAccessToken.cs

```diff
@@ -93,22 +108,97 @@ public void AuthorizeRequest(Action<string, string> authTokenSetter)
 
         void GetOrRenewAuthentication()
         {
-            if (_expiration - DateTime.UtcNow < TimeSpan.FromMinutes(5))
+            if (_expiration - DateTime.UtcNow < TimeSpan.FromMinutes(4))
```

Comment:
@markcowl A comment here about the significance of the 4 minute to expiration time check would be great... 

also what drove the reduction of the one minute here 5 to 4?